### DISCUSSION TITLE
posts/osint/

### DISCUSSION THREAD

**Discussion by giscus** [neospl0it/neospl0it.github.io#1] [Comments]
# posts/osint/

Dive into Open-Source Intelligence (OSINT), understanding its lifecycle, note-taking essentials, and the strategic use of sock puppets in investigations.

https://neospl0it.github.io/posts/osint/

**Comment by neospl0it** [comment]
Nice Blog (Test)

**Comment by voizok** [comment]
Amazing blog ,Thankyou Neo
